### PR TITLE
Reorganise `sinkhorn_stabilized` and `sinkhorn_stabilized_epsscaling`

### PR DESCRIPTION
**Pull request by zsteve**
This PR updates `sinkhorn_stabilized` and `sinkhorn_stabilized_epsscaling` to use the same calling convention as the other `sinkhorn` functions, the convergence criterion, as well as a general refactor.

I think it should be doable to allow batch computations similar to #67 , but those can be added in a separate PR once both this and #67 have been merged.

### CODE REVIEW

**Review by devmotion — COMMENTED** [review]
Thanks, these are some good improvements! I planned to update these functions as well and add some tests :+1: There are still some additional improvements that I have in mind but I think they can be postponed to other PRs. I only made some minor suggestions and I think we should fix the sequence of the epsilons right away (it's incorrect currently).

File: src/OptimalTransport.jl [hunk 1]
```diff
@@ -538,115 +538,137 @@ function sinkhorn_unbalanced2(
...
-            tol=tol,
-            alpha=alpha,
-            beta=beta,
-            return_duals=true,
-            verbose=verbose,
+function sinkhorn_stabilized_epsscaling(μ, ν, C, ε; lambda=0.5, k=5, kwargs...)
+    ε_values = [ε * lambda^(k - j) for j in 1:k]
+    α = zero(μ)
+    β = zero(ν)
+    for ε_i in ε_values
```
[the remark this answers is not in the document]
Comment (zsteve) [reply src/OptimalTransport.jl:555]: Ok thanks, I missed this. (I remember you said you had found a bug, but I wasn't sure where)

File: src/OptimalTransport.jl [hunk 2]
```diff
@@ -538,115 +538,137 @@ function sinkhorn_unbalanced2(
...
-    alpha=zeros(size(mu)),
-    beta=zeros(size(nu)),
+    maxiter=1_000,
+    tol=nothing,
+    atol=tol,
+    rtol=nothing,
+    check_convergence=nothing,
+    alpha_0=nothing,
+    beta_0=nothing,
     return_duals=false,
```
[the remark this answers is not in the document]
Comment (zsteve) [reply src/OptimalTransport.jl:597]: I've now replaced them directly with `alpha = zero(\mu)` and `beta = zero(\nu)`.

File: test/runtests.jl [hunk 3]
```diff
@@ -337,10 +337,25 @@ end
...
+        ν = fill(1 / N, N)
+
+        # create random cost matrix
+        C = pairwise(SqEuclidean(), rand(1, M), rand(1, N); dims=2)
+
+        # compute optimal transport map (Julia implementation + POT)
+        eps = 0.001
+        γ = sinkhorn_stabilized_epsscaling(μ, ν, C, eps; k=5, maxiter=1_000)
+        γ_pot = POT.sinkhorn(μ, ν, C, eps; method="sinkhorn_stabilized", numItermax=5_000)
+        @test γ ≈ γ_pot atol = 1e-6
```
[the remark this answers is not in the document]
Comment (zsteve) [reply test/runtests.jl:358]: Ok that makes sense. Although in this case they're both normalised right?

**Review by davibarreira — APPROVED** [review]

**Pull request merged**